shortcuts.gd

```diff
@@ -27,3 +27,9 @@ func is_quest_active(quest_name: String) -> bool:
 	var quest: Quest = ResourceLoader.load(QUEST_PATH % quest_name, "Quest")
 	if quest == null: return false
 	return QuestSystem.is_quest_active(quest)
+	
+func get_quest_property(id: int, quest_property: String) -> Variant:
+	var quest : Variant = QuestSystem.get_quest_property(id, quest_property)
+	if quest == null: return false
+
+	return quest
```

Comment:
It'd be better to change the variable name to `property` and avoid checking for null, as in a general case it could be that the property we're checking is in fact `null`.



I'd suggest doing something like
```gdscript
func get_quest_property(id: int, quest_property: String) -> Variant:
    return QuestSystem.get_quest_property(id, quest_property)
```